app/models/report.rb

```diff
@@ -25,7 +25,6 @@ def self.inherited(child)
       scoped_search :relation => :environment, :on => :name,  :complete_value => true, :rename => :environment
       scoped_search :relation => :organization, :on => :name, :complete_value => true, :rename => :organization
       scoped_search :relation => :location,    :on => :name,  :complete_value => true, :rename => :location
-      scoped_search :relation => :messages,    :on => :value,                          :rename => :log, :only_explicit => true
       scoped_search :relation => :sources,     :on => :value,                          :rename => :resource, :only_explicit => true
```

Comment:
As I said in the description, I don't find searching for exact log lines useful, I don't know why it is here. Searching for sources is useful, the index is still there.

Reply:
perhaps searching for file name that has changed, however I never used this, does dropping this actually cause any performance gain? or it would be just terribly slow now?

Reply:
When talking to users about this, there were people who explicitly search for things in the logs. I know they told me they searched for exact messages. For example, something failed on one server and they wonder if it also failed on another server. I'm not sure about the source.

Reply:
This search is not for exact log lines, you can search for partial matches as well, e.g. `log ~ error`

Reply:
I will keep this line then, SQL LIKE is still useful. It is explicit only so no big deal.